src/views/JoinHouse.vue

```diff
@@ -65,7 +66,10 @@ export default {
 data(){
     return {
      roomkey: undefined,
-     homename: undefined
+     homename: undefined,
+     rules: {
```

Comment:
Let's give this variable a more specific name. I suggest formRules, but would be happy to hear other thoughts

Reply:
Fixed!